### ISSUE TITLE
Some question about the zero-shot eval in lLVIS.

### ISSUE THREAD

**Issue by MengTanOwn** [IDEA-Research/T-Rex#90]
The paper compares the AP index on LVIS when prompts are 1, 4, 16, 32, and 64. However, there are some categories in LVIS that do not have enough images. How is this situation handled?

**Comment by Mountchicken** [comment]
Hi @MengTanOwn 
It does happen. In this case, we will take as many samples as possible to generate the visual prompt. For example, for class A, there are only 5 samples in LVIS, so we will only take those 5 samples for that class.

**Comment by MengTanOwn** [comment]
Thanks for your answer. Are the visual prompt extracted from the training set or the validation set for LVIS zero-shot?
